tests/test_runs/test_run_functions.py

```diff
@@ -527,10 +527,13 @@ def test_get_run_trace(self):
             flow = openml.flows.sklearn_to_flow(clf)
             flow_exists = openml.flows.flow_exists(flow.name, flow.external_version)
             self.assertIsInstance(flow_exists, int)
+            self.assertGreater(flow_exists, 0)
             downloaded_flow = openml.flows.get_flow(flow_exists)
             setup_exists = openml.setups.setup_exists(downloaded_flow)
             self.assertIsInstance(setup_exists, int)
+            self.assertGreater(setup_exists, 0)
```

Comment:
Maybe we can just use assertGreater instead of also making use of assertIsInstance, since if the argument is False it will throw AttributeError: 'NoneType' object has no attribute 'fail'.